src/Notes.pro

```diff
@@ -41,16 +41,15 @@ HEADERS  += \
 FORMS += $$PWD/mainwindow.ui
 RESOURCES += $$PWD/images.qrc
 
-unix:!macx {
-    isEmpty (PREFIX) {
-        PREFIX = /usr
-    }
-
-    BINDIR = $$PREFIX/bin
-    DATADIR =$$PREFIX/share
-
-    target.path = $$BINDIR
-    INSTALLS += target
+linux:!android {
+    target.path = /usr/bin
+    icon.path = /usr/share/pixmaps
+    desktop.path = /usr/share/applications
+    icon.files += $$PWD/packaging/linux/common/notes.png
+    desktop.files += $$PWD/packaging/linux/common/notes.desktop
+
+    TARGET = notes
+    INSTALLS += target desktop icon
 }
```

Comment:
Not good, you should give the user the choice to install the app wherever he/she wants.
Create a symlink from $$PREFIX/bin/notes to /usr/bin/notes when PREFIX is not /usr